dockerfiles/feature-support/hsm-patches/0001-Introduce-generic-keypair-generation-interface-and-e.patch

```diff
@@ -368,7 +369,32 @@ index ec7f279..57cba43 100644
  
  	/* pubkey attributes */
  	pkcs11_addattr(&pubtmpl, CKA_ID, id, id_len);
-@@ -310,6 +320,105 @@ int pkcs11_generate_key(PKCS11_SLOT_private *slot, int algorithm, unsigned int b
+ 	if (label)
+ 		pkcs11_addattr_s(&pubtmpl, CKA_LABEL, label);
+ 	pkcs11_addattr_bool(&pubtmpl, CKA_TOKEN, TRUE);
+-	pkcs11_addattr_bool(&pubtmpl, CKA_ENCRYPT, TRUE);
```

Comment:
This is for RSA keys? 
I'm not sure if RSA keys can do derive. you can usually use ENCRYPT

Reply:
Removed derive